cypress/e2e/specialFeatures/atiAnalytics/index.cy.js

```diff
@@ -125,6 +125,42 @@ const canonicalTestSuites = [
     useReverb: true,
     tests: [assertPageView],
   },
+  {
+    path: '/dari',
+    runforEnv: ['local'],
+    service: 'dari',
+    pageIdentifier: 'dari.page',
+    applicationType: 'responsive',
+    contentType: 'index-home',
+    useReverb: true,
+    tests: [
+      assertPageView,
+      assertScrollableNavigationComponentView,
+      assertScrollableNavigationComponentClick,
+      assertDropdownNavigationComponentView,
+      assertDropdownNavigationComponentClick,
+      assertMessageBannerComponentView,
+      assertMessageBannerComponentClick,
+    ],
+  },
+  {
+    path: '/dari',
+    runforEnv: ['test'],
+    service: 'dari',
+    pageIdentifier: 'dari.page',
+    applicationType: 'responsive',
+    contentType: 'index-home',
+    useReverb: true,
+    tests: [
+      assertPageView,
+      assertScrollableNavigationComponentView,
+      assertScrollableNavigationComponentClick,
+      assertDropdownNavigationComponentView,
+      assertDropdownNavigationComponentClick,
+      // assertMostReadComponentView,
+      // assertMostReadComponentClick,
```

Comment:
Now that toggles has been fixed & we've decreased the number of items to 5, I believe these might now work...?

Select ATI Analytics tests
```
CYPRESS_APP_ENV=test yarn cypress:interactive
```

```suggestion
      assertMostReadComponentView,
      assertMostReadComponentClick,
```